shell/platform/darwin/ios/BUILD.gn

```diff
@@ -203,9 +203,8 @@ shared_library("ios_test_flutter") {
   # This bug results engine build failure since the engine treats warnings as errors.
   # The `-Wno-unguarded-availability-new` can be removed when the Xcode bug is fixed.
   # See details in https://github.com/flutter/flutter/issues/128958.
```

Comment:
OT: Can this be removed now? The linked issue seems fixed.

Reply:
Answer: definitely seems to be fixed in Xcode 16. Will send a followup and see how it fares on the bots.

Reply:
Sent in https://github.com/flutter/engine/pull/56689